source/documentation/doc-Release_Notes/topics/ref-Technology_Preview_Features_RHV.adoc

```diff
@@ -46,11 +46,13 @@ For details about IPv6 support, see link:{URL_virt_product_docs}{URL_format}admi
 * `virt-v2v` cannot change the default kernel in the GRUB2 configuration. The kernel configured on the guest operating system is not changed during the conversion, even if a more optimal version is available.
 
 * After converting a Debian or Ubuntu virtual machine from VMware to KVM, the name of the virtual machine's network interface may change, and will need to be configured manually
+
 |NVDIMM host devices |Support for attaching an emulated NVDIMM to virtual machines that are backed by NVDIMM on the host machine. For details, see link:{URL_virt_product_docs}{URL_format}virtual_machine_management_guide/index#conc-nvdimm-host-devices_vm_guide_administrative_tasks[NVDIMM Host Devices].
 |Open vSwitch (OVS) cluster type support |Adds Open vSwitch networking capabilities.
 |Shared and local storage in the same data center |Allows the creation of single-brick Gluster volumes to enable local storage to be used as a storage domain in shared data centers.
 |Cinderlib Integration |Leverage CinderLib library to use Cinder-supported storage drivers in {virt-product-fullname} without a Full Cinder-OpenStack deployment. Adds support for Ceph storage along with Fibre Channel and iSCSI storage. The Cinder volume has multipath support on the {hypervisor-fullname}.
```

Comment:
Cinderlib Integration - "Integration should be lower case.
Change "Leverage" to "Leverages" for consistency.